charts/k8ssandra/templates/grafana/stargate.configmap.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if (and .Values.monitoring.grafana.provision_dashboards .Values.stargate.enabled) }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-k8ssandra-grafana-stargate-dashboard
```

Comment:
What do you think about shortening the name to `{{ .Release.Name }}-cassandra-overview`?